tests/tensorflow2/test_keras.py

```diff
@@ -471,11 +465,10 @@ def test_include_regex(out_dir, tf_eager_mode):
         assert tr.tensor(tname).value(0) is not None
 
 
+@pytest.mark.skip_if_non_eager
```

Comment:
Let's see what we can do so all tests can be run at once, without relying on test scaffolding to preserve the environment